rls/build.gradle

```diff
@@ -19,7 +20,26 @@ dependencies {
             project(':grpc-core').sourceSets.test.output  // for FakeClock
 }
 
+javadoc {
+    // do not publish javadoc since it is not published yet
+    exclude 'io/grpc/rls/**'
+}
+
+jacocoTestReport {
+    classDirectories.from = sourceSets.main.output.collect {
+        fileTree(dir: it, exclude: ['**/io/grpc/lookup/**'])
+    }
+}
+
 configureProtoCompilation()
 
 // do not publish 'grpc-rls'
 [publishMavenPublicationToMavenRepository]*.onlyIf { false }
+
+jacocoTestReport {
+    reports {
```

Comment:
Why add this different setting than other subprojects like grpc-core?

Reply:
removed.